manifests/a/AutoDesk/DWGTrueView/4.0.19.0/AutoDesk.DWGTrueView.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: AutoDesk.DWGTrueView
+PackageVersion: 4.0.19.0

```

```suggestion
PackageVersion: 24.2.72.0
```
Version must match control panel